pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -221,10 +222,12 @@ func fillDefaults(instance *operatorv1.LogCollector) []string {
 					v1.SyslogLogDNS,
 					v1.SyslogLogFlows,
 				}
-
 				// Include the field that was modified (in case we need to display error messages)
 				modifiedFields = append(modifiedFields, "AdditionalStores.Syslog.LogTypes")
 			}
+			if len(syslog.Encryption) == 0 {
+				instance.Spec.AdditionalStores.Syslog.Encryption = v1.EncryptionNone
```

Comment:
I think you want to also add the field to the modifiedFields list, like the others are.

Reply:
Done.